Makefile

```diff
@@ -25,3 +26,7 @@ unit-tests: fmt vet ## Run unit tests.
 
 build: fmt vet lint ## Build audit-scanner binary.
 	go build -o bin/audit-scanner .
+
+.PHONY: build-container
+build-container: fmt vet lint ## Build audit-scanner binary.
```

Comment:
Nit:
I'm ok as it is, but shouldn't it depend on `build` target? But in that case we may be better serviced by having the target trigger with the binary filename, and redoing the Makefile.

Reply:
Yes we can do it. I was using the `build-container` to quickly build the container image to be used in the scanner deployment in the cluster. It's not intend to be the official container build command.

Reply:
I've removed the target because it is used only in my local env.